uint/Cargo.toml

```
@@ -4,20 +4,21 @@ homepage = "http://parity.io"
 repository = "https://github.com/paritytech/parity-common"
 license = "MIT/Apache-2.0"
 name = "uint"
-version = "0.8.1"
+version = "0.8.2"
 authors = ["Parity Technologies <admin@parity.io>"]
 readme = "README.md"
 edition = "2018"
 
 [dependencies]
-byteorder = { version = "1", default-features = false }
-rustc-hex = { version = "2.0", default-features = false }
-quickcheck = { version = "0.6", optional = true }
-crunchy = { version = "0.2", default-features = true }
+byteorder = { version = "1.3.2", default-features = false }
+rustc-hex = { version = "2.0.1", default-features = false }
+quickcheck = { version = "0.9.0", optional = true }
+crunchy = { version = "0.2.2", default-features = false }
+rand = { version = "0.7.2", default-features = false }
```

I'd like to avoid dependency on rand, it seems unnecessary, we could use `next_u32` to generate 10/10/80% chances for quickcheck.

---

Alternatively, make rand optional
```suggestion
rand = { version = "0.7.2", default-features = false, optional = true }
```
I guess that would require renaming `quickcheck` dependency and exposing a new feature `quickcheck = ["<quickcheck-renamed>", "rand"]`